ttnn/ttnn/__init__.py

```diff
@@ -15,6 +15,10 @@
 import tt_lib as _tt_lib
 import ttnn._ttnn
 
+import site
+
+site.addsitedir(pathlib.Path(__file__).parent.parent / "tt_lib")
```

Comment:
Is this necessary if we have `tt_lib` available as an importable package now?

Reply:
no, currently don't needs these code. I had deleted it